mapbook-iOS/Cells/LocalPackageCell.swift

```diff
@@ -87,6 +80,11 @@ class LocalPackageCell: UITableViewCell {
             return
         }
         
+        guard AppContext.shared.isUpdatable(package: package) else {
+            SVProgressHUD.showSuccess(withStatus: "\(package.item?.title ?? "The mmpk") is already up to date.")
```

Comment:
Feels like this should be a `showInfo` SVProgressHUD call rather than `showSuccess`.